{
  "name": "babel-await-add-catch",
  "version": "0.0.1",
  "description": "Babel plugin helps automatically add try catch console when encountering await",
  "main": "lib/index.js",
  "scripts": {
    "dev": "node local.test.js",
    "build": "rm -rf lib && npx babel src --out-dir lib",
    "deploy": "npm run build && npm publish"
  },
  "files": [
    "lib"
  ],
  "keywords": [
    "babel-plugin",
    "ast",
    "async",
    "await",
    "try-catch"
  ],
  "author": "dave-wind",
  "license": "ISC",
  "devDependencies": {
    "@babel/cli": "^7.16.8",
    "@babel/core": "^7.17.2",
    "@babel/preset-env": "^7.16.11",
    "@babel/types": "^7.17.0",
    "babel-template": "^6.26.0"
  }
}
